### PR TITLE
[Bug 1143946] Community Hub: Filter by last_contribution_date advanced filter.

### PR DESCRIPTION
**Pull request by mythmon**
This adds a new major UI component to the top contributors page, advanced filters. Advanced filters are more free form, supporting operators like less than or greater than. They can be added or removed, and live in a dialog opened from the top right. It is kind of ugly, but it works, and is pretty quick too. It looks like this:

shot_17063

and

shot_17287

r?

### CODE REVIEW

**Comment by mythmon** [comment]
Uh. Not entirely sure how @AdrianGaudebert's commit got in there. I'll take that out.

**Comment by rlr** [comment]
fancy!

File: kitsune/community/static/js/AdvancedFilters.jsx [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+
+const availableFilters = {
+    'last_contribution_date': {
+        title: 'Days since last contribution',
+        type: 'days_relative',
+        ops: {
+            /* Yes, these are backwards. It is beceause the real data type is
+             * a date, but we are dealing with relative days.
+             * "less than 3 days ago" translates to "date > (now - 3 days)"
+             * The subtraction flips the comparotor.
```
Comment (rlr) [inline kitsune/community/static/js/AdvancedFilters.jsx]: typo "comparotor"?

**Comment by rlr** [comment]
I have a newish db. Gonna do a reindex to play with this.

**Comment by rlr** [comment]
this is awesome :boom: and fast :zap:

**Comment by rlr** [comment]
still waiting on travis :hourglass_flowing_sand:

**Comment by mythmon** [comment]
Travis is having issues today; Linux builds aren't starting. It might be a while until Travis is back with us. http://www.traviscistatus.com/

**Comment by rlr** [comment]
OK. running tests locally then merge

**Pull request merged**